server/src/main/java/org/candlepin/pinsetter/tasks/KingpinJob.java

```diff
@@ -53,7 +55,7 @@
     @Inject protected UnitOfWork unitOfWork;
     @Inject protected Configuration config;
     @Inject private EventSink eventSink;
-
+    @Inject private PinsetterKernel pinsetterKernel;
```

Comment:
Perhaps I missed something, but, why does the kernel get injected here if it isn't used?